### PR TITLE
testing CI, final adjustments, cleaning up code, preparing for release

### LINKED ISSUE #1: Snowflake Minimal VS/MVP
Get a basic VS working with basic SELECT on Snowflake tables.

### PR DESCRIPTION
**Pull request by pj-spoelders**
Closes #1

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "java.codeGeneration.useBlocks": true,
+  "java.saveActions.organizeImports": true,
+  "java.sources.organizeImports.starThreshold": 3,
+  "java.sources.organizeImports.staticStarThreshold": 3,
+  "java.test.config": {
+    "vmArgs": [
+      "-Djava.util.logging.config.file=src/test/resources/logging.properties",
+      "-Dtest.debug='true'",
+      "-Dcom.exasol.dockerdb.image=8.24.0"
```
Comment (kaklakariada) [inline .vscode/settings.json]:
```suggestion
      "-Dcom.exasol.dockerdb.image=8.31.0"
```

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "source.fixAll": "explicit"
+  },
+  "java.codeGeneration.useBlocks": true,
+  "java.saveActions.organizeImports": true,
+  "java.sources.organizeImports.starThreshold": 3,
+  "java.sources.organizeImports.staticStarThreshold": 3,
+  "java.test.config": {
+    "vmArgs": [
+      "-Djava.util.logging.config.file=src/test/resources/logging.properties",
+      "-Dtest.debug='true'",
```
Comment (kaklakariada) [inline .vscode/settings.json]:
```suggestion
      "-Dtest.debug=true",
```

File: README.md
Comment (kaklakariada) [inline README.md:1]: Contains some broken links, e.g. `[Virtual Schemas Repository][virtual-schemas]`

**Review by kaklakariada — COMMENTED** [review]
General: Indentation of Java files is broken for some files. Please run formatter for all files.

**Review by kaklakariada — CHANGES_REQUESTED** [review]
General: Indentation of Java files is broken for some files. Please run formatter for all files.
Also I am not sure if all findings from https://github.com/exasol/snowflake-virtual-schema/pull/4 are implemented here, or do you want to implement them in another PR?

**Comment by pj-spoelders** [comment]
I seem to have missed a bunch of the other review's comments .. working on it.

File: README.md
Comment (pj-spoelders) [reply README.md:1]: Fixed

File: .vscode/settings.json (hunk 2 above)
Comment (pj-spoelders) [reply .vscode/settings.json]: Removed it altogether

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**